libs/dh/shared/feature-authorization/src/lib/dh-selected-actor.component.html

```diff
@@ -14,11 +14,56 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+<div
+  #dropup
+  class="actors-dropup-container watt-elevation"
+  *rxLet="actorGroups$ as actorGroups"
+>
+  <ng-container *ngFor="let actorGroup of actorGroups; let i = index">
+    <div class="watt-label organization-name-header">
+      {{ actorGroup.organizationName }}
+    </div>
+    <div
+      class="actor-container"
+      *ngFor="let actor of actorGroup.actors"
+      (click)="selectActor(actor)"
+    >
+      <div class="actor-container-inner">
+        <div>
+          <div class="watt-label gln-label">{{ actor.gln }}</div>
+          <div class="watt-text-s actor-name-label">
+            {{ actor.actorName }}
+          </div>
+        </div>
+        <watt-icon
+          class="actor-selected-icon"
+          name="checkmark"
+          *ngIf="actor.selected"
+        ></watt-icon>
+      </div>
+    </div>
+    <div
+      class="organization-group-divider"
+      *ngIf="i < actorGroups.length - 1"
+    ></div>
```

Comment:
suggestion: to avoid an empty spacer element, we can wrap each group in a `div` and give it a class (e.g. `actor-group`). We can then use the class to make space between the each group - `.actor-group + .actor-group { ... }`. This selector will apply styling to every `.actor-group` preceded by another `.actor-group`.

Reply:
very cool. fixed :D